highfidelity/hifi
# case 18450: Do not show login dialog if requested not to on the command line.

# Description
https://highfidelity.fogbugz.com/f/cases/18450/Auto-Tester-Test-accounts-are-being-signed-out-of-the-localhost-between-test-runs

An option has been added to the command line parameters.
If `--no-login-suggestion` appears on the command line then "Sign Into High Fidelity" pop up will not appear.
This is required for automatic testing
# Testing
1.  Run interface, logout and close interface.
2.  Run interface from the command line with the `--no-login-suggestion` option and wait 30".
     Ensure that the Login window does not appear.
3.  Logout and quit interface
4.  Run interface from the command line without any options and verify that the Login window appears after about 3".
5.  Check the box "Keep me logged in", login to interface and quit.
6.  Run interface from the command line without any options and verify that:
   a.  you are still logged in
   b.  no Login window appears.
